ui/app/AppLayouts/Profile/LeftTab/Menu.qml

```diff
@@ -28,6 +28,7 @@ ScrollView {
                 source: "../../../img/profile/" + modelData.filename
                 active: profileMenu.profileCurrentIndex === modelData.id
                 Layout.fillWidth: true
+                width: profileMenu.width
```

Comment:
`parent.width` should do the trick as well I think

Reply:
I just checked your solution, but it is still keeping fixed width.
I am wondering `parent.width` will point `profileMenu.width` really.
```
ScrollView ->Column->Repeater->delegate:MenuButton
```

Reply:
Ah right, the `Repeater` needs `width: parent.width` as well

Reply:
It is still not working, I am not sure the right reason.